### ISSUE TITLE
Add a link to organization on community billboards

### ISSUE THREAD

**Issue by pkfrank** [forem/forem#19315]
## Is your feature request related to a problem? Please describe.

As a billboard creator, I want people checking out my billboard to easily be able to click through my (organization) profile page.

## Describe the solution you'd like

For Community billboards, both the organization name and the profile image link to the organization profile.

## Additional context

![arrows pointing to locations that should be linked](https://p-DgFRWr8y.t2.n0.cdn.getcloudapp.com/items/z8uGQDAB/3f3d7059-e996-4f75-8134-bb7472d092ca.jpg?v=2238c62bf9fe90d36028f53778f2a2bd)

**Comment by rt4914** [comment]
@pkfrank For this we might need some suggestions mainly for hover state of icon+name, i.e. , should be show underline or just highlight the content a bit, etc.

**Comment by pkfrank** [comment]
@rt4914 Good point.  For now, let's keep it simple and not introduce any fancy hover state.  Let's use the cursor "pointer" style when hovering either of those links.
